src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -515,6 +519,15 @@ public void swipeLeft(int times) {
         }
     }
 
+    public void longPress(By by) {
+        Actions actions = new Actions(getAppiumDriver());
+        actions.moveToElement(find(by)).clickAndHold().perform();
+    }
+
+    public void longPress(String id) {
+        longPress(PageUtil.buildBy(configuration, id));
+    }
+
```

Comment:
For consistency sake, I think I'd like a method that accepts a `WebElement` as a parameter.

```
public void longPress(WebElement element) {
        Actions actions = new Actions(getAppiumDriver());
        actions.moveToElement(element).clickAndHold().perform();
    }
```
```
public void longPress(By by) {
    longPress(waitForElement(by));
}
```